Python/Product/Uwp/visualstudio_py_remote_launcher.py

```diff
@@ -25,7 +25,7 @@
 import os
 import sys
 import ptvsd
-import visualstudio_py_debugger as vspd
+from ptvsd import visualstudio_py_debugger as vspd
```

Comment:
`import ptvsd.visualstudio_py_debugger as vspd`

(Shouldn't be any different, but it's the more common style of writing this)